src/ucp/wireup/ep_match.h

```diff
@@ -14,7 +14,13 @@
 /**
  * Maximal value for EP connection sequence number
  */
-#define UCP_EP_MATCH_CONN_SN_MAX    ((ucp_ep_match_conn_sn_t)-1)
+#define UCP_EP_MATCH_CONN_SN_MAX ((ucp_ep_match_conn_sn_t)-2)
+
+
+/**
+ * Reserved value indicating invalid EP connection sequence number
+ */
+#define UCP_EP_MATCH_CONN_SN_INVALID ((ucp_ep_match_conn_sn_t)-1)
```

Comment:
why need ? maybe UCP_EP_MATCH_CONN_SN_MAX and MAX would be the same?

Reply:
unhandled

Reply:
removed